oldwinapi.nimble

```diff
@@ -1,6 +1,6 @@
 [Package]
 name          = "oldwinapi"
-version       = "1.0.0"
+version       = "1.0.1"

```

Comment:
Nice that you update the version, but it's a breaking change and so should be "1.1.0". :-)

Reply:
Actually it should be "2.0.0".